code/modules/integrated_electronics/subtypes/manipulation.dm

```diff
@@ -304,19 +304,28 @@
 	activators = list("pulse in" = IC_PINTYPE_PULSE_IN,"pulse out" = IC_PINTYPE_PULSE_OUT)
 	spawn_flags = IC_SPAWN_RESEARCH
 	power_draw_per_use = 50
-	var/max_w_class = WEIGHT_CLASS_NORMAL
 	var/max_items = 10
-
 /obj/item/integrated_circuit/manipulation/grabber/do_work()
+	var/max_w_class = assembly.w_class
```

Comment:
Allows ttv grabbing and fully bypassing the ttv nerf as almost no one is going to search every drone.

Reply:
@KorPhaeron

Reply:
last time i tried blowing up TTVs inside of drones nothing happened ( they nullspaced or something )

Reply:
ok it should no longer be able to grab TTVs